### ISSUE TITLE
video.duration.timestamp is empty

### ISSUE THREAD

**Issue by NekoSuneVR** [talmobi/yt-search#84]
there pose be a data for endpoint `video.duration.timestamp` and `video.timestamp`

but its gives me a empty data even description is empty.

i checked output all i get is this but no timestamp.

not sure if this a Linux Server issue or only works for windows

this running on Linux Server

![image](https://github.com/user-attachments/assets/09528158-cc2a-4a06-a0ae-c73562907115)

**Comment by talmobi** [comment]
@NekoSuneVR yes you are right -- this issue was discovered a few days ago -- implementing a fix right now

**Comment by talmobi** [comment]
@NekoSuneVR hello please try the new version `npm install yt-search@2.12.0` -- genre/category data is unavailable and video description data is limited.

**Comment by talmobi** [comment]
@NekoSuneVR hello please try patched version `npm install yt-search@2.12.1`
